src/pages/Footer/Footer.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import './Footer.scss';
+
+function Footer() {
+  return (
+    <footer>
+      <div className="footerInner">
+        <div className="footerMenu">
+          <div className="footerList">
+            <h3>Product</h3>
+            <ul>
+              <li>Running</li>
+              <li>Soccer</li>
+              <li>Basketball</li>
+            </ul>
+          </div>
+          <div className="footerList">
+            <h3>Account</h3>
+            <ul>
+              <li>Login</li>
+              <li>Join</li>
+              <li>Cart</li>
+            </ul>
+          </div>
```

Comment:
```suggestion
{FOOTER_LIST.map(({ id, title, list }) => {
            return (
              <div className="footerList" key={id}>
                <h3>{title}</h3>
                <ul>
                  {list.map(text => {
                    return <li key={text}>{text}</li>;
                  })}
                </ul>
              </div>
            );
          })}
```

해당 부분은 상수데이터를 만들어서 위와 같이 구현할 수 있겠습니다!
객체 안에 배열 값을 다시 map을 사용하는 방식으로 하면 되겠습니다!